using-ide-vscode.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: page
+title: Developing Flutter apps in an IDE
+
+permalink: /using-ide-vscode/
+---
+
+<ul class="tabs__top-bar">
+    <li class="tab-link" data-tab-href="/using-ide/">Android Studio / IntelliJ</li>
+    <li class="tab-link current">VS Code</li>
+</ul>
+
+<div class="tabs__content current" markdown="1">
+
+The Dart extension provides a fully integrated development experience in Visual Studio Code.
+
+* TOC Placeholder
+{:toc}
+
+## Installation and setup
+
+Please follow the [Editor Setup](/get-started/editor/) instructions to install
+the Dart extension (which includes Flutter functionality).
+
+### Updating the extension<a name="updating"/>
+
+Updates to the extension will be shipped on a regular basis. By default, VS Code will
+automatically update extensions when updates are available.
+
+To install updates manually:
+
+1. Click on the Extensions button in the side bar
```

Comment:
Isn't it 'sidebar' ?

https://en.wikipedia.org/wiki/Sidebar_(computing)

Reply:
Hmm, VS calls it "Side Bar"... WDYT?

<img width="279" alt="screen shot 2018-04-05 at 08 44 23" src="https://user-images.githubusercontent.com/1078012/38353180-9596c944-38ad-11e8-9bc7-97939efd38fd.png">

Reply:
Oh, yes they do: https://code.visualstudio.com/docs/getstarted/userinterface

But I think we should use caps then -- Side Bar -- to be consistent.

Reply:
Done!